plugins/rolesum.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: rolesum
+spec:
+  shortDescription: Summarize RBAC roles for subjects
+  homepage: https://github.com/Ladicle/kubectl-rolesum
+  version: v1.4.0
+  platforms:
+    - uri: https://github.com/Ladicle/kubectl-rolesum/releases/download/v1.4.0/kubectl-rolesum_linux-amd64.tar.gz
+      sha256: "aaa1780cdef58cb7711a323af7d78bf8ab1b637ebab2bab9fd969dcfc8514a79"
+      bin: kubectl-rolesum
+      files:
+        - from: ./kubectl-rolesum_linux-amd64/kubectl-rolesum
+          to: "."
+        - from: ./kubectl-rolesum_linux-amd64/LICENSE
+          to: "."
+      selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+    - uri: https://github.com/Ladicle/kubectl-rolesum/releases/download/v1.4.0/kubectl-rolesum_darwin-amd64.tar.gz
+      sha256: "f5fc2ab889bb0ddf47bc9e467eb6517ea7c876f7262a016d0658e15fd11aa366"
+      bin: kubectl-rolesum
+      files:
+        - from: ./kubectl-rolesum_darwin-amd64/kubectl-rolesum
+          to: "."
+        - from: ./kubectl-rolesum_darwin-amd64/LICENSE
+          to: "."
+      selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+    - uri: https://github.com/Ladicle/kubectl-rolesum/releases/download/v1.4.0/kubectl-rolesum_windows-amd64.tar.gz
+      sha256: "deb3a8eab654cd95ce93c246aa6b570fa41ece1a4f321404d037aef2534565db"
+      bin: kubectl-rolesum
+      files:
+        - from: ./kubectl-rolesum_windows-amd64/kubectl-rolesum
+          to: "."
```

Comment:
I am guessing you didn't test this but this won't work.

For something to be executable on windows, it needs an .exe prefix. (you need to add it to `to:` and `bin:` fields).

Reply:
fix: 1b6a46f